src/PresentationalComponents/Email/Email.js

```diff
@@ -121,7 +121,9 @@ const Email = () => {
                                             <Skeleton size='lg'></Skeleton>
                                         )}
                                     </FlexItem>
-                                    <a href='#'>Not correct?</a>
+                                    <a href={ `https://www.${insights.chrome.isBeta ? 'qa.' : ''}redhat.com/wapps/ugc/protected/personalInfo.html` }>
```

Comment:
You should also include `rel="noopener noreferrer" target="_blank"` to `a` element to open this in new tab.